lib/iris/experimental/ugrid/__init__.py

```diff
@@ -45,11 +45,11 @@
 from ...util import guess_coord_axis
 
 __all__ = [
-    "CFUGridReader",
```

Comment:
NOTE: I don't think this belongs or is needed in public API, so I removed it.
It is imported (in deferred usage) within netcdf.py, but I don't think that warrants including here :  once the two are re-integrated, that would become a local reference.